server/src/database/migrations/sql/1_initial_schema.up.sql

```diff
@@ -89,6 +89,7 @@ create table board_sessions
     "connected"  boolean      not null DEFAULT false,
     "ready"      boolean      not null DEFAULT false,
     raised_hand  boolean      not null DEFAULT false,
+    "views_shared_note"   boolean not null DEFAULT false,
```

Comment:
Please create a new migration script for anything that you want to add to the database schemas. Modifying an existing schema won't work and can break the database.